engine/access/rpc/backend/backend_scripts.go

```diff
@@ -128,7 +128,10 @@ func (b *backendScripts) executeScript(
 
 	case IndexQueryModeFailover:
 		localResult, localDuration, localErr := b.executeScriptLocally(ctx, scriptRequest)
-		if localErr == nil || isInvalidArgumentError(localErr) || status.Code(localErr) == codes.Canceled {
+		// Note: temporarily ignoring error types and falling back to ENs for ALL errors
```

Comment:
Can you link to an issue or PR to indicate when this can be switched back

Reply:
it's already switch on master/v0.33. this is just here to avoid backporting the changes since we're upgrading in 2 weeks